README.md

```diff
@@ -278,6 +278,7 @@ Currently, the following package managers are supported:
 * [Cargo](https://doc.rust-lang.org/cargo/) (Rust)
 * [Conan](https://conan.io/) (C / C++, *experimental* as the VCS locations often times do not contain the actual source
   code, see [issue #2037](https://github.com/oss-review-toolkit/ort/issues/2037))
+* [Carthage](https://github.com/Carthage/Carthage) (iOS / Cocoa)
```

Comment:
Carthage need to be above Conan as list is alphabetically sorted on package manager name